samples/XAMLCatalog/XAMLCatalog/UIKitControls/UILabelViewController.h

```diff
@@ -17,5 +17,30 @@
 
 #import "MenuTableViewController.h"
 
-@interface UILabelViewController : MenuTableViewController
+@interface UILabelViewController : UIViewController <UITextFieldDelegate>
+@property (nonatomic) UITextField* text;
```

Comment:
Makes sense but could you have added that support into TestEnabledUITextField. That way we can all benefit from the addition. This additional is like useful functionality so we wouldn't need to use sentinel characters to denote the end of input etc.  



---
In reply to: [105038372](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r105038372) [](ancestors = 105038372,104793428)